src/Server/Resource/DoctrineResource.php

```diff
@@ -86,6 +87,17 @@ class DoctrineResource extends AbstractResourceListener implements
      */
     protected $hydrator;
 
+    /** @var InstantiatorInterface|null */
```

Comment:
As I mention before, please change it to 3-line comment, to be consistent in the file:

```php
/**
 * @var InstantiatorInterface|null
 */
```